platform/pulp/tasking/tasks.py

```diff
@@ -225,14 +237,24 @@ def apply_async(self, *args, **kwargs):
 
         :param group_id:    The id that identifies which group of tasks a task belongs to
         :type group_id:     uuid.UUID
+        :param args:          The positional arguments to pass on to the task.
+        :type  args:          tuple
+        :param kwargs:        The keyword arguments to pass on to the task.
+        :type  kwargs:        dict
+        :param **options:     For all options accepted by apply_async please visit:
+                              http://docs.celeryproject.org/en/latest/reference/celery.app.task.html#celery.app.task.Task.apply_async  #NOQA
 
         :return:            An AsyncResult instance as returned by Celery's apply_async
         :rtype:             celery.result.AsyncResult
         """
-        tag_list = kwargs.pop('tags', [])
-        group_id = kwargs.pop('group_id', None)
-        async_result = super(UserFacingTask, self).apply_async(*args, **kwargs)
-        async_result.tags = tag_list
+        async_kwargs = {
+            "args": args,
+            "kwargs": kwargs,
+            "options": options
```

Comment:
options isn't an keyword argument, it's a dict of keywork arguments that should be an unpacked before being passed in to apply_async

`async_result = super(UserFacingTask, self).apply_async(args=args, kwargs=kwargs, **options)`